cfgen/dhall/types/ClusterDesc.dhall

```diff
@@ -28,7 +28,7 @@ let Node =
   { hostname : Text
   , data_iface : Text
   , data_iface_type: Optional ./Protocol.dhall
-  , m0_servers : List M0Server
+  , m0_servers : Optional (List M0Server)
```

Comment:
@andriytk You have to reflect this change in [`ShowSchema.__call__()`](https://github.com/Seagate/cortx-hare/blob/411bffff2c279fd131872e70f5b71a90adac9505/cfgen/cfgen#L78) and [18/CFGEN](https://github.com/Seagate/cortx-hare/tree/411bffff2c279fd131872e70f5b71a90adac9505/rfc/18#cluster-description-file-cdf).

Also don't forget to increment [`__version__`](https://github.com/Seagate/cortx-hare/blob/411bffff2c279fd131872e70f5b71a90adac9505/cfgen/cfgen#L22).

Reply:
Done. Thank you @vvv for reviewing this!